### DISCUSSION TITLE
Core Definition under Fair Source 100

### DISCUSSION THREAD

**Discussion by sanieldalib** [cirruslabs/tart#433] [Q&A]
Hey team! Thank you for the continued work and support of Tart. I'm especially thankful for the relicensing under Fair Source 100 as this allows companies with a blanket ban on AGPL to use Tart.

I was wondering if you could clarify the definition of a core under the new license. Are both performance and efficiency cores counted? I know that Anka only considers performance cores for licensing (https://docs.veertu.com/anka/licensing/).

Could you clarify this in the license and documentation?

**Comment by fkorotkov** [comment]
Glad that the license change enables your organization to enjoy Tart!

Interesting that Anka differentiates cores, from a Virtual Machine perspective there are only virtual cores. @neoswallow did some empirical  experiments [here](https://github.com/cirruslabs/tart/discussions/396#discussioncomment-4940899) and it seems you can have two VMs running and one can use all the performance cores and the other can use all the efficient cores if the second workload is not CPU intensive.

Therefore, from licensing perspective it's not reasonable to differentiate. Clarifying it in #432. PTAL.

**Comment by neoswallow** [comment]
It looks like Tart's Fair Source 100 description is clearer than Anka's version with that change.
Even though they stated that they only count P cores, they only gave the example on M1 Mac mini.
They specifically noted to contact them for M1Max/Ultra core pricing.
Not to mention actual core pricing info is not available publicly.
But then that last part is same for Tart, too.
So even though actual pricing comparison would make it more clear, both products opted not to show actual licensing cost publicly.
